app/templates/client/app/app(coffee).coffee

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-angular.module '<%= scriptAppName %>', [<%= angularModules %>]
+angular.module '<%= scriptAppName %>', [<%- angularModules %>]
```

Comment:
Yeoman is now using ejs instead of lodash. By default ejs escapes html characters, such as quotes. The minus sign returns values unescaped.